src/components/Dashboard/styles.js

```diff
@@ -0,0 +1,70 @@
+import { makeStyles } from "@material-ui/styles";
+
+export default makeStyles((theme) => ({
+    dividerRoot: {
+        marginBottom: theme.spacing(3),
+    },
+    footer: {
+        width: "100%",
+        height: 128, // This is needed to get the vertical scrolling to stop cutting off the bottom of the content.
+
+        [theme.breakpoints.down("sm")]: {
+            height: 32,
+        },
+    },
+    section: {
+        marginTop: theme.spacing(2),
+        width: "100%",
+    },
+    sectionNews: {
+        width: "66%",
+        [theme.breakpoints.down("md")]: {
+            width: "100%",
+        },
+    },
+    sectionEvents: {
+        width: "33%",
+        overflowWrap: "break-word",
+        overflow: "hidden",
+        [theme.breakpoints.down("md")]: {
+            width: "100%",
+        },
+    },
+    sectionItems: {
+        display: "flex",
+        flexWrap: "wrap",
+        justifyContent: "flex-start",
+        paddingRight: 0,
+
+        // Try to eek as much space out of the iPhone SE cards as possible.
+        [theme.breakpoints.down("md")]: {
+            padding: 0,
+        },
+    },
+    subtitle: {
+        marginBottom: theme.spacing(2),
+    },
+    gridRoot: {
+        overflow: "auto", // Needed for vertical scrolling.
+        height: "90vh", // Needed to get the vertical scrolling working.
```

Comment:
Not something to change now, but so far in my experience, this tends to work better as a calculation from 100% (like 100vh - someOtherSize). Otherwise that 10% gets bigger and bigger for larger screens.  Figuring out the right calculation though is not as easy as it seems it should be.